### DISCUSSION TITLE
SVG

### DISCUSSION THREAD

**Discussion by jindal1979** [dotnet/maui#7014] [Q&A]
The SVG appicon and SVG images are truing to dark black with RC3 release. Is anyone else facing the same issues.

**Comment by JeffIPV** [comment]
Noticing the same. #7115 #7101 
Seems the work around is to not use style in the svg to apply the colors. And if your svg is complicated its a painstaking process to go through.
I ended up going back to RC2 since my project uses a lot of svgs.
Ran cmd prompt as administrator and ran `dotnet workload install maui --from-rollback-file https://aka.ms/dotnet/maui/rc.2.json --source https://aka.ms/dotnet6/nuget/index.json --source https://api.nuget.org/v3/index.json --include-previews`
